src/Migrator.php

```diff
@@ -77,6 +78,11 @@ public function getMigrationNameLimit(): ?int
         return $this->migrationNameLimit = $limit;
     }
 
+    public function getMaxSqlOutputLength(): int
```

Comment:
Why need getter?

Reply:
I can remove if you think no need.

Reply:
Seems, no. `maxSqlOutputLength` need in `MigrationBuilder` that create into `Migrator` itself.